frontend/src/app/modules/login/containers/login-screen/login-screen.component.spec.ts

```diff
@@ -1,20 +1,19 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { LoginScreenContaineromponent } from './login-screen.component';
 
-import { LoginScreenComponent } from './login-screen.component';
-
-describe('LoginScreenComponent', () => {
-  let component: LoginScreenComponent;
-  let fixture: ComponentFixture<LoginScreenComponent>;
+describe('LoginScreenContainerComponent', () => {
+  let component: LoginScreenContaineromponent;
+  let fixture: ComponentFixture<LoginScreenContaineromponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ LoginScreenComponent ]
+      declarations: [ LoginScreenContaineromponent ]
```

Comment:
Seems you want to named it LoginScreenContainerComponent

Reply:
yes, it was renamed. 
Container -> it's SMART  -> It's top level component of module that uses another components/directives
Comonent -> It's Dump -> aka directive